src/number/format-decimal.js

```diff
@@ -14,18 +14,24 @@ module.exports = function(value, localeData, options) {
 	// round to desired precision
 	value = Math.abs(Math.round(value * precisionScaling) / precisionScaling);
 
-	var integerValue = value | 0;
+	var integerValue = Math.trunc( value );
```

Comment:
Or use `Math.floor()`, which is supported everywhere. This is forced to be a positive value anyway via `Math.abs()`, so negative rounding cases aren't a concern.